action.yml

```diff
@@ -1,13 +1,42 @@
-name: 'Wait'
-description: 'Wait a designated number of milliseconds'
+name: 'Swift test action for Datadog'
+author: 'Datadog, Inc.'
+description: 'Automatically instruments and runs your tests reporting to Datadog CI Visibility'
 inputs:
-  milliseconds:  # id of input
-    description: 'number of milliseconds to wait'
+  api_key:
+    description: "Datadog API key"
     required: true
-    default: '1000'
-outputs:
-  time: # output will be available to future steps
-    description: 'The current time after waiting'
+  application_key:
+    description: "Datadog Application key"
+    required: true
```

Comment:
is requiring this a good idea? We only need it if ITR is used, right?

Reply:
I think it simplifies the onboarding, they just fill two values instead of one, I have the same in the standard Swift documentation

Reply:
yeah ok that makes sense